yarn.lock

```diff
@@ -5814,7 +5814,7 @@
   resolved "https://registry.yarnpkg.com/@tabler/icons-react/-/icons-react-2.47.0.tgz#b704e7ae98f95be8bd6e938b4b2e84cd20b0cf31"
   integrity sha512-iqly2FvCF/qUbgmvS8E40rVeYY7laltc5GUjRxQj59DuX0x/6CpKHTXt86YlI2whg4czvd/c8Ce8YR08uEku0g==
   dependencies:
-    "@tabler/icons" "2.32.0"
+    "@tabler/icons" "2.47.0"
```

Comment:
Unrelated, the `yarn.lock` file wasn't up to date. Will clean this up when it's time to merge.